controllers/item-controller.js

```diff
@@ -0,0 +1,54 @@
+const { Category, Game, Item } = require('../models')
+
+const itemController = {
+  getItems: async (req, res, next) => {
+    try {
+      const items = await Item.findAll()
+      res.json({
+        status: 'success',
+        data: items.toJSON()
```

Comment:
`items` is already an array of objects, `items.toJSON()` would show an error: `TypeError: items.toJSON is not a function`.

Reply:
fixed!

Reply:
updated README.md and .env.example docs as well